TCIABrowser/TCIABrowser.py

```diff
@@ -902,6 +902,11 @@ def addSelectedToDownloadQueue(self):
     self.downloadQueue = {}
     refSeriesList = []
     rows = [i.row() for i in self.seriesTableWidget.selectionModel().selectedRows(0)]
+    # Limiting how many series or images that can be downloaded at once
+    if len(rows) > 20 or self.imagesToDownloadCount > 1000:
+        message = "You are trying to download more than 20 series or 1,000 combined images, please download them in seperate batches."
```

Comment:
To me, this limit seems artificial, but I have not tried downloading this much using TCIABrowser. Maybe testing experience of @SL2027 showed that this is not practical?

Reply:
Just a reminder that there are 2 options.  Download + Index just downloads and puts the data into the DICOM database.  Download + Load will download it, put it in the DICOM database and load it into the scene.  This is more what I'm concerned about.  Can you load 20+ series into a scene without causing problems?  I guess we should try some tests, but wondered if others already had a rough idea.

Reply:
> Can you load 20+ series into a scene without causing problems?

There is no single answer. It will depend on how large are the series, how much RAM you have on the computer, what is the kind of those series.

Reply:
Yeah, I think based on Andrey's feedback a warning is a better idea if we do anything at all.  I'm still not sure what would make sense as a good threshold if we did the warning, which has me leaning towards abandoning this idea.

Reply:
Do you think we can add a setting that has a default value of 20, but the users can always change this warning threshold much like what we did with the storage folder setting?

Reply:
I'm concerned that it'll just introduce an extra layer of annoyance for a lot of people if the threshold isn't set properly by default. Now they'd have to routinely click away extra warnings or go change a setting that they didn't need to mess with before.

What if we always do a pop-up after they click one of the download buttons saying, "You've selected X series consisting of Y megabytes/gigabytes of data to download." and then give them OK/Cancel buttons to proceed or bail out.  This way people will get a summary of what they're about to download/open and should be able to draw their own conclusion about whether they have enough disk space or robust enough hardware to deal with that.